### PR TITLE
[PHPStan Baseline] A few small fixes

### PR DESCRIPTION
**Pull request by AlexandruGG**
This PR:

* [x] Fixes a few more issues from the PHPStan baseline

### CODE REVIEW

File: tests/static-analysis/random.php [hunk 1]
```diff
@@ -38,27 +38,19 @@ function random_checkStringMap(CollectionInterface $collection): void
 }
 
 $intGenerator = static function (): Generator {
-    while (true) {
```
Comment (AlexandruGG) [inline tests/static-analysis/random.php:41]: for now `while (true)` is a new PHPStan error: https://github.com/phpstan/phpstan/issues/5917.

These ones don't matter for us though, makes no difference for the static analysis checks

Comment (drupol) [reply tests/static-analysis/random.php:41]: Nice !

**Review by drupol — APPROVED** [review]
Feel free to merge this even if the CS is not green, I'll fix it a bit later, I'm definitely burried with work.

**Comment by AlexandruGG** [comment]
@drupol thanks for the CS fixer config change 💯

**Pull request merged**